docs/eventing/sources/rabbitmq-source/README.md

```diff
@@ -0,0 +1,219 @@
+# Creating a RabbitMQ Source
+
+This topic describes how to create a RabbitMQ Source.
+
+## Prerequisites
+
+To use the RabbitMQ Source, you must have the following installed:
+
+1. [Knative Eventing](../../../install/yaml-install/eventing/install-eventing-with-yaml.md)
+1. [RabbitMQ Cluster Operator (optional)](https://github.com/rabbitmq/cluster-operator) - our recommendation is [latest release](https://github.com/rabbitmq/cluster-operator/releases/latest)
+1. [CertManager v1.5.4](https://github.com/jetstack/cert-manager/releases/tag/v1.5.4) - easiest integration with RabbitMQ Messaging Topology Operator
+1. [RabbitMQ Messaging Topology Operator](https://github.com/rabbitmq/messaging-topology-operator) - our recommendation is [latest release](https://github.com/rabbitmq/messaging-topology-operator/releases/latest) with CertManager
+
+## Install the RabbitMQ controller
+
+1. Install the RabbitMQ Source controller by running the command:
+
+    ```bash
+    kubectl apply -f {{ artifact(org="knative-sandbox", repo="eventing-rabbitmq", file="rabbitmq-source.yaml") }}
+    ```
+
+1. Verify that `rabbitmq-controller-manager` and `rabbitmq-webhook` are running:
+
+    ```bash
+    kubectl get deployments.apps -n knative-sources
+    ```
+
+    Example output:
+
+    ```{ .bash .no-copy }
+    NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+    rabbitmq-controller-manager     1/1     1            1           3s
+    rabbitmq-webhook                1/1     1            1           4s
+    ```
+
+## Create a RabbitMQ cluster (optional)
+
+1. Deploy a RabbitMQ cluster:
+
+    1. Create a YAML file using the following template:
+
+        ```yaml
+        apiVersion: rabbitmq.com/v1beta1
+        kind: RabbitmqCluster
+        metadata:
+          name: <cluster-name>
+          annotations:
+            # A single RabbitMQ cluster per Knative Eventing installation
+            rabbitmq.com/topology-allowed-namespaces: "*"
+        ```
+        Where `<cluster-name>` is the name you want for your RabbitMQ cluster,
+        for example, `rabbitmq`.
+
+    1. Apply the YAML file by running the command:
+
+        ```bash
+        kubectl create -f <filename>
+        ```
+        Where `<filename>` is the name of the file you created in the previous step.
+
+1. Wait for the cluster to become ready. When the cluster is ready, `ALLREPLICASREADY`
+will be `true` in the output of the following command:
+
+    ```bash
+    kubectl get rmq <cluster-name>
+    ```
+    Where `<cluster-name>` is the name you gave your cluster in the step above.
+
+    Example output:
+
+    ```{ .bash .no-copy }
+    NAME          ALLREPLICASREADY   RECONCILESUCCESS   AGE
+    rabbitmq      True               True               38s
+    ```
+
+For more information about configuring the `RabbitmqCluster` CRD, see the
+[RabbitMQ website](https://www.rabbitmq.com/kubernetes/operator/using-operator.html).
+
+## Reference an external RabbitMQ Cluster
```

Comment:
same comment as earlier, seems like maybe this content or the headings are duplicated? If this needs to be included in both places maybe make it a snippet instead so that it's not duplicated content being maintained in 2 places?